src/druidRequester.ts

```diff
@@ -462,8 +462,11 @@ export function druidRequesterFactory(parameters: DruidRequesterParameters): Ply
 
                   response
                     .pipe(new Combo({ packKeys: true, packStrings: true, packNumbers: true }))
+                    .on('error', streamError)
                     .pipe(rowBuilder)
-                    .pipe(stream, { end: false });
+                    .on('error', streamError)
+                    .pipe(stream, { end: false })
+                    .on('error', streamError);
```

Comment:
Need to remove the last `.on(error)` actually, otherwise we get a nice little infinite recursion
```suggestion
```